### PR TITLE
Replace "You attempting" with "You are attempting"

### PR DESCRIPTION
**Pull request by radekholy24**

### CODE REVIEW

**Comment by ridlees** [comment]
Yes, it would work. 

However, as I said, the message was changed to: 
`Your are not allowed to send this request with 'from' = 498728123212"`

=> which should be `you are not allowed`.

**Comment by yet-another-radek** [comment]
@ridlees, do I understand correctly that you are suggesting this change?
```diff
- You attempting to use a number that is not assigned to your account
+ You are attempting to use a phone number that you are not allowed
```
?

**Comment by ridlees** [comment]
No, the Error message has been changed on production. So, we don't need to introduce new change.

**Comment by radekholy24** [comment]
@ridlees, sorry, I am totally confused. Can you please tell me what should I do with this PR? Should I discard it or leave it as it is now or change it somehow (how?)?

**Comment by ridlees** [comment]
The error message has changed on production. So, we are (at least from my POV) updating old thing. So, I would suggest.

**Review by ridlees — COMMENTED** [review]
What about this? The Sender configuration instead of number (which is not number in cases of WeChat, Viber and Telegram but ID) and then on channel basis description what Sender configuration means in their context (ID or phone number).

File: conversations/v3/openapi.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ components:
             title: Constraint Violation
 
     Forbidden:
-      description: You attempting to use a number that is not assigned to your account
+      description: You are attempting to use a number that is not assigned to your account
```
[the remark this answers is not in the document]
Comment (radekholy24) [reply conversations/v3/openapi.yaml:76]: @ridlees, this one is only:
```suggestion
      description: You are attempting to use a sender configuration that is not assigned to your tyntec account
```
right?

**Review by ridlees — APPROVED** [review]

File: conversations/v3/openapi.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ridlees) [reply conversations/v3/openapi.yaml:76]: yup,

**Pull request merged**